### ISSUE TITLE
Build error with export

### ISSUE THREAD

**Issue by jafri** [uuosio/ascdk#46]
@learnforpractice could you kindly have a look at this:

https://github.com/uuosio/ascdk/blob/6621212b34d7b2c76208d6e1b77dda0902e1c97c/examples/atomicassets/atomicassets.contract.ts#L8

Building fails if `export` is added

Works fine without export

**Comment by learnforpractice** [comment]
What's the error?

**Comment by jafri** [comment]
You can try with master branch, lots of errors

**Comment by learnforpractice** [comment]
I haven't seen any errors. Here is the test result:

```
test.py::test_hello PASSED
test.py::test_action PASSED
test.py::test_counter PASSED
test.py::test_token PASSED
test.py::test_codegeneration PASSED
test.py::test_notify PASSED
test.py::test_singleton PASSED
test.py::test_allow PASSED
test.py::test_escrow PASSED
test.py::test_balance PASSED
```

**Comment by learnforpractice** [comment]
Ok, let me update the source code and have another try.

**Comment by learnforpractice** [comment]
Why do you want to export `KvContract`? This is the main class and does not expect to be exported.

**Comment by jafri** [comment]
Good for contract composability, like contract `Abc` importing `KvContract` to instantiate it, and then call it's methods

Also another dev messaged me saying they were running into an issue and I narrowed it down to the `export`, so its unexpected behavior for `export` to throw issue in valid code

**Comment by learnforpractice** [comment]
Yeah, sounds like the asc compiler does not happy with the exported class. Give me some the to find out the reason.

**Comment by jafri** [comment]
It kind of looks like our transformer misses the exported class

**Comment by learnforpractice** [comment]
Fixed by 2fba770d70ff7e2c3ae84ba52aa2438c86b74085
